docs/installation.md

```
@@ -63,23 +63,28 @@ INSTALLED_APPS = [
     # Django Control Room (list after panels so they appear in one section)
     'dj_control_room',
-    
+
     # Your apps
     'myapp',
     # ...
 ]
 ```
 
+> **Note:** `dj_control_room_base` is installed automatically as a dependency of `dj-control-room`, but it **must** also be listed in `INSTALLED_APPS` so Django can discover its template tag library (`dcr_icons`) and static assets. If you forget it, you will see a `'dcr_icons' is not a registered tag library` error and Django's system check will report `dj_control_room.E001`.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)